frontend/packages/ux-editor/src/components/config/ConditionalRenderingComponent.tsx

```diff
@@ -304,75 +306,74 @@ class ConditionalRendering extends React.Component<IConditionalRenderingComponen
                   (key: any) => {
                     const paramName = key;
                     return (
-                      <div className='row align-items-center mb-1' key={key}>
-                        <div className='col-3 col'>
-                          <div className='form-group a-form-group mt-1 disabled'>
-                            <label className='a-form-label' htmlFor={paramName}>
-                              {
-                                this.props.language[
-                                  'ux_editor.modal_configure_conditional_rendering_configure_input_param_helper'
-                                ]
+                      <>
+                        <label className={classes.label} htmlFor={paramName}>
+                          {
+                            this.props.language[
+                              'ux_editor.modal_configure_conditional_rendering_configure_input_param_helper'
+                            ]
+                          }
+                        </label>
+                        <div className={classes.configureInputParamsContainer} key={key}>
+                          <input
```

Comment:
I think this component needs total re-factoring. The CSS-fix within this PR is just because we need the modal to look great if we want to release studio next week.